vendor/knative.dev/pkg/hack/update-deps.sh

```diff
@@ -24,7 +24,7 @@ set -o pipefail
 cd ${ROOT_DIR}
 
 # Ensure we have everything we need under vendor/
-dep ensure
+dep ensure $@
```

Comment:
Did you mean to put this in this PR?

Reply:
it's just dep update generated and hack scripts didn't remove it, so I guess it's intentional :)